raiden/tests/integration/network/transport/rif_comms/test_client.py

```diff
@@ -0,0 +1,123 @@
+import pytest
+
+from raiden.tests.integration.network.transport.rif_comms.node import Node as CommsNode, Config as CommsConfig
+from raiden.tests.integration.network.transport.utils import generate_address
+from transport.rif_comms.utils import notification_to_payload
+
+
+@pytest.fixture()
+@pytest.mark.parametrize("amount_of_nodes")
+def comms_nodes(amount_of_nodes) -> {int, CommsNode}:
+    nodes = {}
+
+    # setup
+    for i in range(1, amount_of_nodes + 1):
+        node = CommsNode(CommsConfig(i))
+        nodes[i] = node
+
+    yield nodes
+
+    # teardown
+    for node in nodes.values():
+        node.stop()
+
+
+@pytest.mark.parametrize("amount_of_nodes", [1])
+def test_locate_own_peer_id(comms_nodes):
+    comms_node = comms_nodes[1]
+
+    # attempt to locate self
+    assert comms_node.client._get_peer_id(comms_node.address) is not ""
+
+
+# FIXME: causes ERR_NO_PEERS_IN_ROUTING_TABLE in comms node although it passes
+@pytest.mark.parametrize("amount_of_nodes", [1])
+def test_locate_unregistered_peer_id(comms_nodes):
+    comms_node = comms_nodes[1]
+
+    # attempt to locate an unregistered peer
+    assert comms_node.client._get_peer_id(generate_address()) is ""
+
+
+# FIXME: comms node prints strange ServerUnaryCall message
```

Comment:
documentado en _RIF Comms: pending issues_

Reply:
it's a minor squabble on my part. this print is humanly unreadable, so i am not sure what purpose it serves.